### PR TITLE
Master fix

### PR DESCRIPTION
**Pull request by cqy123456**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [x] Code Refactoring

**Which branch you want to cherry-pick to?**

Not Available

**Which issue(s) this PR fixes:**

Fixes #

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: core/src/db/DBImpl.cpp [hunk 1]
```diff
@@ -68,13 +68,13 @@ DBImpl::DBImpl(const DBOptions& options)
...
 
-    Start();
+    DBImpl::Start();
 }
 
 DBImpl::~DBImpl() {
     ConfigMgr::GetInstance().Detach("storage.auto_flush_interval", this);
 
-    Stop();
+    DBImpl::Stop();
```
Comment (cydrain) [inline core/src/db/DBImpl.cpp:77]: this->Stop();

File: core/src/db/insert/MemManagerImpl.cpp [hunk 2]
```diff
@@ -131,6 +131,7 @@ MemManagerImpl::ValidateChunk(int64_t collection_id, const DataChunkPtr& chunk)
 
                 break;
             }
+            default: {}
```
Comment (cydrain) [inline core/src/db/insert/MemManagerImpl.cpp:134]: default:
    break;

File: core/src/server/grpc_impl/GrpcRequestHandler.cpp [hunk 3]
```diff
@@ -275,7 +275,7 @@ CopyDataChunkToEntity(const engine::DataChunkPtr& data_chunk,
         if (data == nullptr || data->data_.empty())
             continue;
 
-        auto single_size = data->data_.size() / id_size;
+        auto single_size = (id_size != 0) ? (data->data_.size() / id_size) : (id_size);
```
Comment (cydrain) [inline core/src/server/grpc_impl/GrpcRequestHandler.cpp:278]: auto single_size = (id_size != 0) ? (data->data_.size() / id_size) : 0;

File: core/src/segment/Segment.cpp [hunk 4]
```diff
@@ -67,6 +67,7 @@ Segment::AddField(const std::string& field_name, DataType field_type, int64_t fi
             real_field_width = field_width;
             break;
         }
+        default: {}
```
Comment (cydrain) [inline core/src/segment/Segment.cpp:70]: default: break;

**Review by cydrain — APPROVED** [review]

**Review by shengjun1985 — APPROVED** [review]

**Review by wxyucs — APPROVED** [review]

**Pull request merged**